Update to the grid widget (#2485)
umbraco/UmbracoDocs: Umbraco-StarterKit/Widgets/Grid/index.md

# The Grid

> The Grid Widget lets you set up a framework of widgets, you will have the ability to make it look the way u want. You can do so by setting up the general look of the grid, and choose what widgets you want your grid to contain.

Reviewer: ```suggestion
The Grid Widget lets you set up a framework of widgets and you will have the ability to make it look the way you want. 

When you add the Grid widget you get to set up the general look of the grid, and choose what widgets you want it  to contain.
```

## Columns

> A Column is the boxes wherein you create your content when working in a grid. Most of the grid widgets require you to set a column width. This is because it sets the width for how big the box will be and by that determine how many consecutive Columns can be in a row.

Reviewer: Instead of "most of the grid widgets", I think it would be best to say "all" - even though it currently isn't a requirement on all, it really should be.

```suggestion
Columns are the boxes wherein you create your content when working in a grid. Most of the grid widgets require you to set a column width. This is because it sets the width for how big the box will be and by that determine how many consecutive columns can be in a row.
```

## Rows

> A Row is a kind of shelf for the columns in the grid, each row has a total column width of 100%. Say you wanted three columns on your row each of these would have to have a column width of 33%.

Reviewer: ```suggestion
A Row is a kind of shelf for the columns in the grid, and each row has a total column width of 100%. Say you wanted three columns on your row each of these would have to have a column width of 33%.
```

![Rows and Columns](images/Rows-Columns.png)

## Using the Grid

It can be very beneficial to be using a grid because it makes sure that everything stays in the alignment that you have set up for the grid.

> The grid is awesome for multiple reasons, it keeps your page-aligned beautifully, however that is not all a grid can do, the grid also provides you a lot of awesome tools such as Navigation or Line break.

Reviewer: ```suggestion
The grid is good for multiple reasons. It keeps your page-aligned beautifully and also provides you with a lot of great tools such as Navigation or Line break.
```

- Background Color - This lets you add a background color to the entire grid.
- Text Color - This lets you change the text color in your grid.
- Animate - This gives your grid an animation that makes it fade into view when the user arrives at the grid.

![Grid Settings](images/Grid-Settings.png)

## Presentation widgets

These widgets are tools to make your grid as nice look as possible, each of them offers unique functionality, that in some way has the potential to improve the overall look of your grid.

- [Header](Header/index.md)
- [Text](Text/index.md)
- [Navigation](Navigation/index.md)
- [Line break](Line-break/index.md)
- [Sub grid](Sub-grid/index.md)
- [Slider](Slider/index.md)
- [Button](Button/index.md)

## Content widgets

These widgets are tools to help you get your desired content shown in the best possible way, each of them offering unique functionalities to display content.

- [Accordion](Accordion/index.md)
- [Quote](Quote/index.md)
- [Code](Code/index.md)
- [Price list](Price-list/index.md)
- [Price table](Price-table/index.md)
- [Umbraco Form](Umbraco-Form/index.md)
- [Pod](Pod/index.md)
- [Card](Card/index.md)

## Media widgets

> These are great tools for adding your media to your grid in a manageable fashion, it is straight forward to use, please note that video files can only be MP4.

Reviewer: ```suggestion
These are great tools for adding media to your grid in a manageable fashion. Please note that video files needs to be MP4.
```
